staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go

```diff
@@ -772,16 +773,16 @@ func TestAuditIDHttpHeader(t *testing.T) {
 		expectedHeader bool
 	}{
 		{
-			"no http header when there is no audit",
+			"http header when there is no audit",
 			"",
 			auditinternal.LevelNone,
-			false,
+			true,
```

Comment:
so this explicitly changed, right? Is the argument that we want to have an id in any case, even when we don't write audit events?

Reply:
A comment about the high level reason would be helpful